v2/emailpassword/common-customizations/sessions/multiple-api-endpoints.mdx

```diff
@@ -112,6 +116,10 @@ Whilst the `cookieDomain` can start with a leading `.`, the value of the `apiDom
 For local development, you should not set the `cookieDomain` to an IP address based domain, or `.localhost` - browsers will reject these cookies. Instead, you should [alias `localhost` to a named domain and use that](https://superuser.com/questions/152146/how-to-alias-a-hostname-on-mac-osx). 
 :::
 
+:::important
```

Comment:
I think this should be moved above the snippet or merged into the important section on the top of the page.
Also, it feels like it's missing information:
- changing the cookieDomain can cause elevated request numbers even if olderCookieDomain is set right
- if olderCookieDomain is not set, it'll lock people with older sessions out
- if olderCookieDomain is set wrong it causes an infinite refresh loop (we might not want to mention this though)
- you only need to set it if you have previous sessions.

Some of these are kind of like warnings, so consider separating those into a danger/warning admonition (if we use those elsewhere). What we have here is kind of an explanation section.

Reply:
If you choose to add a separate step for setting `olderCookieDomain`, then the warnings can go to the top of that, with the explanation (what/how it causes issues) moved into normal text or a `note` admonition.

Reply:
I have updated the PR to add a separate step for setting `olderCookieDomain` and added some warning/info as well. Please take a look